release-notes-v0.3.0.md

```diff
@@ -0,0 +1,34 @@
+v0.3.0
+
+### New features
+
+This release contains a reference implementation of [FIDO IoT draft spec](https://fidoalliance.org/specs/fidoiot/FIDO-IoT-spec-v1.0-wd-20200730.html). This implementation inherits the initial code from the protocol-next branch of [PRI](https://github.com/secure-device-onboard/pri) repository.
+
+* The core protocol implementation supports "list of crypto algorithms", "list of key-exchange protocols".
+* Protocol-samples are added to help developers validate core protocol implementation. These samples
+use hard-coded values.
+* Component-samples for manufacturer, reseller, rendezvous service, owner and device are provided. These samples can be used to realize different use-case scenarios.
+* REST end-points are provided to perform necessary operations on component databases to realize various use-cases.
+
+### Changes to existing features
+
+None  
+
+### Discontinued features
+
+None  
+
+### Fixed Issues
+
+None
+
+### Known Issues
```

Comment:
Please include these 2 defects as known issues,

DE2436 | [PRI-FIDO][Protocol   Sample] Warnings when \mvn exec:java\ command is executed for protocol sample   components 
DE2434 | [Owner]Error Observed while trying to add 14MB file to Owner_ServiceInfo Table.

Reply:
Issues related to mvn exec -> Documented in https://github.com/secure-device-onboard/pri-fidoiot/README.md

Issue regarding serviceinfo file upload -> Updated

Reply:
@trbehera , Can we also say that Client-Intel suppports only EPID not On-Die / ECDSA.

Reply:
This is not a supported opensource platform.